src/gwkokab/inference/jenks.py

```
@@ -0,0 +1,318 @@
+
+
+__all__ = ["pad_and_stack"]
+
+T = TypeVar("T", bound=Array | np.ndarray | int)
+
+
+def _loss(bucket: Sequence[int]) -> float:
+    r"""Calculate the loss for a given bucket of data.
+
+    .. math::
+        \text{loss} = \frac{n \cdot \max(\mathrm{bucket}) - \text{sum}(\mathrm{bucket})}{n \cdot \max(\mathrm{bucket})} \times 100
```

Pylint (3.3.7)

[convention] Line too long (130/100)

(C0301)